tests/unit/stream_alert_alert_processor/test_outputs/test_carbonblack.py

```diff
@@ -50,11 +48,20 @@ class TestCarbonBlackOutput(object):
 
     def setup(self):
         """Setup before each method"""
+        self._mock_s3 = mock_s3()
+        self._mock_s3.start()
+        self._mock_kms = mock_kms()
+        self._mock_kms.start()
```

Comment:
I've had to do this before as well, not sure why. It does seem to be the more stable solution